src/app_engine/join_page.py

```diff
@@ -200,3 +200,90 @@ def post(self, room_id):
         result['is_initiator'])
     logging.info('User ' + client_id + ' joined room ' + room_id +
                  ' with state ' + result['room_state'])
+
+
+class AckPage(webapp2.RequestHandler):
```

Comment:
I'm normally the first person to move things to new files, but the code in this is so closely related to the join code that I thought it made sense to keep it here. That said, if we add more ack messages, it won't make sense to have it here. Which way do you think we should go?